plugins/inputs/radius/sample.conf

```
@@ -0,0 +1,11 @@
+[[inputs.radius]]
+  ## An array of Server IPs and ports to gather from. If none specified, defaults to localhost.
+  servers = ["127.0.0.1:1812","hostname.domain.com:1812"]
+
+  ## Credentials for radius authentication.
+  # username = "myuser"
+  # password = "mypassword"
+  # secret = "mysecret"
```

Those are always required I guess? If so uncomment them to indicate it for the user
```suggestion
  username = "myuser"
  password = "mypassword"
  secret = "mysecret"
```